s/primdata.ss

```diff
@@ -1429,6 +1429,7 @@
   (fxvector [sig [(fixnum ...) -> (fxvector)]] [flags alloc cp02 safeongoodargs])
   (fxvector->list [sig [(fxvector) -> (list)]] [flags alloc safeongoodargs])
   (fxvector-copy [sig [(fxvector) -> (fxvector)]] [flags alloc safeongoodargs])
+  (fxvector-copy! [sig [(fxvector sub-length fxvector sub-length sub-length) -> (void)]] [flags true])
```

Comment:
is there documentation on what all these flags mean?

Reply:
The flags are defined in https://github.com/cisco/ChezScheme/blob/main/s/cmacros.ss#L1871 but not all of them are explained. 

`true` means that the result is always true, so 

`(if (fxvector-copy! ...) x y)`

is reduced in `cp0` to 

`(begin (fxvector-copy! ...) x)`

Now it's an automatic flag, so the `(void)` result in the signature implies this flag, so `true` is not necessary but not harmful.